src/Domain/TransactionService.php

```diff
@@ -136,9 +141,18 @@ public function __construct(
             $baseCurrency
         );
 
-        $this->repayUrl = strval($repayUrl);
-        $this->authCode = strval($authCode);
+        $this->merchantAccount = $merchantAccount;
         $this->recToken = $recToken ? new CardToken($recToken) : null;
+        $this->authCode = strval($authCode);
+        $this->repayUrl = strval($repayUrl);
+    }
+
+    /**
+     * @return string
+     */
+    public function getMerchantAccount(): string
```

Comment:
Please remove `string` type hint. This feature was release at php 7, but this library works with php >= 5.3